application/common/tests/test_utilities.py

```diff
@@ -4,66 +4,28 @@
 from aws_lambda_powertools.utilities.data_classes.sqs_event import SQSRecord
 
 from pytest import raises
-
+from application.common import utilities
```

Comment:
Why does need to be imported? Surely `from ..utilities import` does the same

Reply:
That's needed for the patch on line 55 which use the utiliites module as an object patch